src/test/java/uk/gov/justice/digital/service/DomainDefinitionClientServiceTest.java

```diff
@@ -0,0 +1,153 @@
+package uk.gov.justice.digital.service;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.*;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.digital.config.JobParameters;
+import uk.gov.justice.digital.domain.DomainExecutor;
+import uk.gov.justice.digital.domain.model.DomainDefinition;
+import uk.gov.justice.digital.service.DomainDefinitionClientService;
+import uk.gov.justice.digital.service.DomainService;
+
+import java.io.IOException;
+import java.util.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+
+class DomainDefinitionClientServiceTest {
+
+
+    private static DomainDefinitionClientService domainDefinitionService;
```

Comment:
Better to declare these as private. You can also make use of the `@Mock` annotation to create your mock instances.